AUTHORS

```diff
@@ -2,22 +2,23 @@ OpenRA wouldn't be where it is today without the
 hard work of many contributors.
 
 The OpenRA developers are:
-    * Chris Forbes (chrisf)
+    * Gustas Kažukauskas (PunkPun)
     * Lukas Franke (abcdefg30)
     * Matthias Mailänder (Mailaender)
     * Paul Chote (pchote)
```

Comment:
You can bump me down too for completeness.

Reply:
Done.